Services/DataX.Flow/DataX.Utility.ServiceCommunication/InterServiceCommunicator.cs

```diff
@@ -0,0 +1,144 @@
+﻿// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+using DataX.Contract;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataX.Utility.ServiceCommunication
+{
+    /// <summary>
+    /// Concrete implementation of inter-service communication client.
+    /// </summary>
+    public class InterServiceCommunicator
+    {
+        private const string _ReverseProxyPort = "19081";
```

Comment:
does this have to be hardcoded?

Reply:
Yes this is the port that has been specified for the reverse proxy. Gateway uses the same.